java/src/main/java/com/scalar/admin/AdminCommand.java

```diff
@@ -59,7 +68,34 @@ public static void main(String[] args) {
 
   @Override
   public Integer call() {
-    RequestCoordinator coordinator = new RequestCoordinator(srvServiceUrl);
+    RequestCoordinator coordinator;
+
+    if (srvServiceUrl != null && addresses != null) {
```

Comment:
```suggestion
    if ((srvServiceUrl == null && addresses == null) || (srvServiceUrl != null && addresses != null)) {
```
to simplify the error handling (and to remove duplicate error messages)?